www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -980,17 +973,14 @@ function (declare, domClass, dom, style, fx, Toggler, on, query, Memory,
 
       //--- Show or hide review status message box ---//
 
-      this.showOrHideReviewStatusMessageBox(report.review.comment);
+      this.showReviewStatusMessageBox(report.review.comment);
     },
 
     /**
      * Show or hide the review status message box if the review status is
      * unreviewed.
      */
-    showOrHideReviewStatusMessageBox : function (message) {
-      if (this.reportData.review.status !== ReviewStatus.UNREVIEWED)
-        domClass.remove(this._reviewComment, 'hide');
-      else
+    showReviewStatusMessageBox : function (message) {
```

Comment:
In the bug tree we can change between bugs. You hide this icon every case when this function is called.
We have to add the hide class only if the author is null, so nobody changed the review status ever. This function should be replaced by this one and change the function name back to `showOrHideReviewStatusMessageBox`:
```js
showOrHideReviewStatusMessageBox : function (message) {
  if (this.reportData.review.author)
    domClass.remove(this._reviewComment, 'hide');
  else
    domClass.add(this._reviewComment, 'hide');
}
```
E.g:
- You are on an unreviewed bug. You change the status. In this case you should show this icon (remove hide class).
- You are on a reviewed bug. You click on a different bug on the left tree which was unreviewed. In this case you should hide this icon (add hide class).